x/leverage/handle_lend.go

```diff
@@ -0,0 +1,17 @@
+package leverage
```

Comment:
Do we need this file?

Reply:
It was just a connection from handler.go to the module, so they'll be deleted together